clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -49,3 +49,97 @@ export function createFileWithGivenPath(folderPath: string, fileName: string, pa
     }
 
 }
+
+/**
+ * SettingsService provides read/write configurstion settings functionality
+ */
+export class SettingsService {
+    /**
+     * Get list of local subroutine path
+     * @returns a list of local subroutine path
+     */
+    public static getSubroutineLocalPath(): string[] {
+        return vscode.workspace.getConfiguration().get(SETTINGS_SUBROUTINE_LOCAL_KEY);
+    }
+
+    /**
+     * Get copybook local path based on program file name
+     * @param cobolProgramName is a program file name
+     * @returns a list of local path
+     */
+    public static getCopybookLocalPath(cobolProgramName: string): string[] {
+        const pathList: string[] = vscode.workspace.getConfiguration(SETTINGS_CPY_SECTION).get(PATHS_LOCAL_KEY);
+        return SettingsService.evaluateVariable(pathList, "program_name", cobolProgramName);
```

Comment:
How about evaluating other paths to be consistent? Like subroutines, DSN's, etc.

Reply:
Make sense, but it is rather tricky, because we do not have program name everywhere we want

Reply:
It can be done separately.